PocketApp.swift

```diff
@@ -9,9 +9,11 @@ import SwiftUI
 struct PocketApp: App {
     @UIApplicationDelegateAdaptor var delegate: PocketAppDelegate
 
+    private let dependencyContainer = PocketAppDependencyContainer()
```

Comment:
the use of `PocketAppDependencyContainer` is optional at this stage, since we don't do much injection from here. We can keep it if we feel that we might get rid of the singleton at some point